okd-golang-builder.Dockerfile

```
@@ -0,0 +1,24 @@
+FROM quay.io/centos/centos:stream9
+
+ARG GOPATH
+
+ENV container=oci \
+    GOPATH=${GOPATH:-/go} \
+    GOMAXPROCS=8
+
+RUN mkdir -p /go/src/ && \
```

```suggestion
RUN mkdir -p /go/src/ && \
         yum clean metadata
```
to make sure we expire cached metadata and fetch new repos on every build

---

It doesn't seem to do anything (`yum install` automatically does that)

---

I think the cache would've been removed in the stream9 base already (adding it here does not remove any files)